deployment/ansible/deploy.yml

```diff
@@ -208,14 +198,8 @@
         connection: local
         tags: [health-check]
 
-      - name: Read app version
-        command: "cat {{ project_root }}/VERSION"
-        register: version_output
-        tags: [datadog-event]
-
-      - name: Read app revision
-        command: "cat {{ project_root }}/REVISION"
-        register: revision_output
+      - name: Refresh epoch facts
+        setup:
```

Comment:
I see this is needed as node may have just been deployed by previous steps in same playbook. (I refreshed my mind by reading [past comment](https://github.com/aeternity/infrastructure/pull/59#discussion_r190185260).)